c++/CMakeLists.txt

```diff
@@ -208,43 +208,40 @@ install(FILES
 #install CapnProtoMacros for CapnProtoConfig.cmake build directory consumers
 configure_file(cmake/CapnProtoMacros.cmake cmake/CapnProtoMacros.cmake COPYONLY)
 
-if(NOT MSVC)  # Don't install pkg-config files when building with MSVC
-  # Variables for pkg-config files
-  set(prefix "${CMAKE_INSTALL_PREFIX}")
-  set(exec_prefix "") # not needed since we use absolute paths in libdir and includedir
-  set(libdir "${CMAKE_INSTALL_FULL_LIBDIR}")
-  set(includedir "${CMAKE_INSTALL_FULL_INCLUDEDIR}")
-  set(PTHREAD_CFLAGS "-pthread")
-  set(STDLIB_FLAG)  # TODO: Unsupported
-
-  set(CAPNP_PKG_CONFIG_FILES
-    pkgconfig/kj.pc
-    pkgconfig/capnp.pc
-    pkgconfig/capnpc.pc
-  )
-
-  if(NOT CAPNP_LITE)
-    list(APPEND CAPNP_PKG_CONFIG_FILES
-      pkgconfig/kj-async.pc
-      pkgconfig/kj-gzip.pc
-      pkgconfig/kj-http.pc
-      pkgconfig/kj-test.pc
-      pkgconfig/kj-tls.pc
-      pkgconfig/capnp-rpc.pc
-      pkgconfig/capnp-websocket.pc
-      pkgconfig/capnp-json.pc
-    )
-  endif()
+# Variables for pkg-config files
+set(prefix "${CMAKE_INSTALL_PREFIX}")
+set(exec_prefix "") # not needed since we use absolute paths in libdir and includedir
+set(libdir "${CMAKE_INSTALL_FULL_LIBDIR}")
+set(includedir "${CMAKE_INSTALL_FULL_INCLUDEDIR}")
+set(PTHREAD_CFLAGS "-pthread")
```

Comment:
If the libraries were built with MSVC, I would not expect them to depend on pthread. So it seems odd to me that we'd tell dependents to set `-pthread` when linking with MSVC-built libraries.

But, if these pkg-config files work for you locally, there's probably no harm, either.

Reply:
I understood.
I will set up an MSVC if-statement regarding this flag.

Reply:
Thanks